utils/deployment-utils/deployment_utils.go

```diff
@@ -188,3 +188,10 @@ func ConstructKeylessTransaction(
 	}
 	return contractCreationTxBytes, senderAddress, contractAddress, nil
 }
+
+// AlterNicksTransaction increments the gas price of the transaction used to deploy the contract.
```

Comment:
I think this would be better in the testing utils, since we wouldn't ever use this function in a real deployment.